azext_iot/central/providers/v1/device_provider_v1.py

```diff
@@ -58,12 +59,15 @@ def get_device(self, device_id, central_dns_suffix=CENTRAL_ENDPOINT) -> Device:
 
         return device
 
-    def list_devices(self, central_dns_suffix=CENTRAL_ENDPOINT) -> List[Device]:
+    def list_devices(
+        self, central_dns_suffix=CENTRAL_ENDPOINT,
+    ):
```

Comment:
Why remove the method return type hint rather than update it?

Reply:
fixed